packages/consent-manager/components/dialog.js

```diff
@@ -145,14 +163,20 @@ export default class ConsentPreferences extends Component {
               this.handleFold(name)
             }}
             aria-label={
-              this.state.showCategories[name] ? 'See less' : 'See more'
+              this.state.showCategories[name]
+                ? `See less in ${name}`
+                : `See more in ${name}`
+            }
+            aria-expanded={this.state.showCategories[name] ? `true` : `false`}
+            aria-controls={
+              this.state.showCategories[name] ? categoryListID : null
```

Comment:
`null` because we only render the contents of the dropdown if the dropdown is open.